Packages/Registration/Routines/DPTLK.m

```diff
@@ -38,7 +41,8 @@ S DIC("W")=$S($D(DIC("W")):DIC("W"),1:"")
  K DPTIFNS,DPTS,DPTSEL
  S DPTCNT=0
  ; -- Check input for format an length
- G CHKDFN:DPTX?1A!(DPTX'?.ANP)!($L(DPTX)>30)&('$G(DGVIC40))
+ ; G CHKDFN:DPTX?1A!(DPTX'?.ANP)!($L(DPTX)>30)&('$G(DGVIC40)) ; OSE/SMH - Allow one character input for CJK languages
```

Comment:
indicating this is the old line maybe helpful (and consistent with comments in other files in this commit)